uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -187,11 +187,11 @@ sub IsCandidateInfoValid {
         #1) Check to see if the file is of type DICOM###########
         #2) Check to see if the header matches the patient-name#
         ########################################################
-        if ( ( $_ ne '.' ) && ( $_ ne '..' ) ) {
+        if ( ( $_ ne '.' ) && ( $_ ne '..' ) && (basename($_) ne '.DS_Store')) {
```

Comment:
Thanks @gluneau 

but:
1) we are looping over files (not directories), I think
and
2)  what is returned  from $File::Find::name is the fullpath (including /tmp/ImagingUpload..../ at the beginning of the filename) 
so I feel we are always going to satisfy the if statement for the conditions on '.' and '..'